### PR TITLE
[prometheus-kafka-exporter] new chart donation

### PR DESCRIPTION
**Pull request by gkarthiks**
#### What this PR does / why we need it:
A new chart donation to prometheus-community

#### Checklist
- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name

@prometheus-community/helm-charts-admins Please invite @golgoth31 to `prometheus-community/helm-charts-maintainers`

/cc @scottrigby I have created a fresh PR incorporating all the changes requested in the review.

### CODE REVIEW

**Comment by gkarthiks** [comment]
Also deprecated the chart in my repo https://github.com/gkarthiks/helm-charts/tree/master/charts/prometheus-kafka-exporter#prometheus-kafka-exporter

File: charts/prometheus-kafka-exporter/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+maintainers:
+  - name: gkarthiks
+    email: github.gkarthiks@gmail.com
+  - name: golgoth31
+    email: golgoth31@notrenet.com
+engine: gotpl
+annotations:
+  "artifacthub.io/links": |
+    - name: Chart Source
+      url: https://gkarthiks.github.io/helm-charts/charts/prometheus-kafka-exporter
```
Comment (torstenwalter) [inline charts/prometheus-kafka-exporter/Chart.yaml:21]: You can update it to this repoy

**Review by torstenwalter — DISMISSED** [review]

**Comment by torstenwalter** [comment]
From my side this looks good. Once @golgoth31 was added to this repository he can do another review and merge this.

**Comment by scottrigby** [comment]
Also linking previous PRs for comment history:
- https://github.com/prometheus-community/helm-charts/pull/91
- https://github.com/prometheus-community/helm-charts/pull/184

**Comment by golgoth31** [comment]
Hi,
@gkarthiks I see that the TLS patch is not part of this PR. Is it normal ?

**Comment by gkarthiks** [comment]
aah, sorry about that @golgoth31 . Pushed with the new commit. Please review.

**Review by golgoth31 — APPROVED** [review]
OK for me

**Review by torstenwalter — APPROVED** [review]

**Pull request merged**